independent-publisher-connectors/Google Photos/apiDefinition.swagger.json

```diff
@@ -0,0 +1,161 @@
+        "operationId": "CreateAlbum",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1/albums/%7B%7BalbumId%7D%7D": {},
+    "/v1/albums/%7B%7Balbum.id%7D%7D": {},
+    "/v1/albums/%7B%7BalbumId%7D%7D:addEnrichment": {},
+    "/v1/albums/%7B%7BalbumId%7D%7D:share": {},
+    "/v1/albums/%7B%7BalbumId%7D%7D:unshare": {},
+    "/v1/albums/%7B%7BalbumId%7D%7D:batchAddMediaItems": {},
+    "/v1/albums/%7B%7BalbumId%7D%7D:batchRemoveMediaItems": {},
+    "/v1/mediaItems": {
+      "get": {
+        "summary": "List Media Items",
+        "description": "\nList all media items from a user's Google Photos library.\n\n### HTTP request\n\n`GET https://photoslibrary.googleapis.com/v1/mediaItems`\n\nThe URL uses [gRPC Transcoding](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) syntax.\n\n### Query parameters\n\n|Parameters|||\n|-|-|-\n|`pageSize`|`integer`|Maximum number of media items to return in the response. Fewer media items might be returned than the specified number. The default `pageSize` is 25, the maximum is 100.|\n`pageToken`|`string`|A continuation token to get the next page of the results. Adding this to the request returns the rows after the `pageToken`. The `pageToken` should be the value returned in the `nextPageToken` parameter in the response to the `listMediaItems` request.|\n\n### Request body\n\nThe request body must be empty.\n\n### Response body\n\nIf successful, the response body contains data with the following structure:\n\nList of all media items from the user's Google Photos library.\n\nJSON representation\n\n```json\n{\n  \"mediaItems\": [\n    {\n      object (MediaItem)\n    }\n  ],\n  \"nextPageToken\": string\n}\n```\n\n|Fields|||\n|-|-|-\n|`mediaItems[]`|`object` ([MediaItem](https://developers.google.com/photos/library/reference/rest/v1/mediaItems#MediaItem))|Output only. List of media items in the user's library.\n|`nextPageToken`|`string`|Output only. Token to use to get the next set of media items. Its presence is the only reliable indicator of more media items being available in the next request.\n\n### Authorization Scopes\n\nRequires one of the following OAuth scopes:\n\n*   `https://www.googleapis.com/auth/photoslibrary`\n*   `https://www.googleapis.com/auth/photoslibrary.readonly`\n*   `https://www.googleapis.com/auth/photoslibrary.readonly.appcreateddata`\n",

```

Comment:
@juliamuiruri4 the description will show up as the operation description when selecting the information button. We suggest keeping it short to fit in the text box. No period required if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)